servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -254,12 +256,18 @@ public synchronized Throwable fillInStackTrace() {
 
         @Override
         public String toString() {
-            return "HttpResponseException{ metaData=" + metaData.toString(DEFAULT_HEADER_FILTER) + '}';
+            return "HttpResponseException{" +
+                    "metaData=" + metaData +
+                    ", message='" + message + '\'' +
```

Comment:
Consider using `super.toString() + ", metaData=" + metaData` to preserve the original exception formatting